backend/app/usecase/repository/url_fake.go

```diff
@@ -47,7 +47,7 @@ func (u URLFake) GetByAlias(alias string) (entity.URL, error) {
 
 // GetByAliases finds all URL for a list of aliases
 func (u URLFake) GetByAliases(aliases []string) ([]entity.URL, error) {
```

Comment:
What if we have named function parameters? So instead of ([]entity.URL, error) we'd have (urls []entity.URL, err error)

Reply:
@arberiii Let's open a new issue to track this large scale refactor!